.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,44 @@
+name: Scorecard supply-chain security
+on:
+  branch_protection_rule:
+  schedule:
+    - cron: '16 9 * * 2'
```

Comment:
Now that's a funny-looking cron. crontab.guru says
> “At 09:16 on Tuesday.” 

OK fine.

Reply:
Oddly specific 😆